nextpnr-xilinx/meta.yaml

```diff
@@ -22,19 +22,22 @@ requirements:
     - {{ compiler('c') }}
     - {{ compiler('cxx') }}
     - python {{ python }}
+    - libboost
+    - py-boost
     - yosys
+    - pypy3.6
```

Comment:
The dependency on `pypy3.6` worries me a little. Let's merge and log an issue to follow up on that.